GMessenger/Messenger.Database/Configurations/AttachmentConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Messenger.Database.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Messenger.Database.Configurations
+{
+    internal class AttachmentConfiguration : IEntityTypeConfiguration<Attachment>
+    {
+        public void Configure(EntityTypeBuilder<Attachment> builder)
+        {
+            builder.Property(a => a.Url).HasMaxLength(500);
+
+            builder
+             .HasOne(b => b.Message)
+             .WithMany(a => a.Attachments)
+             .HasForeignKey(b => b.MessageId);
+        }
```

Comment:
kazkaip builder formatavimas visur skirtingas 🤔 not sure kaip teisingiausia formatuot (ar is karto po builder kelt i kita eilute ar ne), bet manau atstumai visur turetu but vienodi islaikyti